src/MonitorProfiler/Environment/Environment.h

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma once
+
+#include "corhlpr.h"
+#include "tstring.h"
+
+/// <summary>
+/// Interface allowing getting and setting of environment variables.
+/// </summary>
+DECLARE_INTERFACE(IEnvironment)
```

Comment:
I'm ok if you keep this, but do we really need an interface here? Consider just folding all the environment classes into one utility class that always uses the profiler.

Reply:
I wanted to keep how the environment is read separate from an inherent dependency on the profiler so that way we have the possibility of unit testing different classes without having to mock an `ICorProfilerInfo*` implementation. With the way it is, tests can implement an `IEnvironment` to pass in default environment values and read out any changes that were made during testing.